simpeg/objective_function.py

```diff
@@ -463,6 +465,8 @@ def __call__(self, m, f=None):
             else:
                 objective_func_value = objfct(m)
             fct += multiplier * objective_func_value
+            obj_vals.append(objective_func_value)
+        self._last_obj_vals = obj_vals
```

Comment:
This is what I'm referring to when I say I don't particularly like this approach, but at least it's a private variable and only used in the similarity printers.